test/python/importer/jit_ir/ivalue_import/submodules.py

```diff
@@ -7,7 +7,7 @@
 import torch
 from torch_mlir.dialects.torch.importer.jit_ir import ModuleBuilder
 
-# RUN: %PYTHON %s | torch-mlir-opt | FileCheck %s
+# RUN: %PYTHON %s | torch-mlir-opt%exeext | FileCheck %s
```

Comment:
The python path might need some extra handling in the lit files. I am not sure if it's configured correctly, but in the past we've run into issues where the default configured path does not support installations that contain spaces (which happens on Windows). Have a look at how this is handled for llvm/lldb/clang (I think mlir is not correct)